nerfstudio/models/splatfacto.py

```diff
@@ -738,7 +714,7 @@ def get_outputs(self, camera: Cameras) -> Dict[str, Union[torch.Tensor, List]]:
             1,
             quats_crop / quats_crop.norm(dim=-1, keepdim=True),
             viewmat.squeeze()[:3, :],
-            projmat.squeeze() @ viewmat.squeeze(),
+            None,  # Deprecated projmat
```

Comment:
If https://github.com/nerfstudio-project/gsplat/pull/149 gets merged then this needs to be updated, I think just changing the API all at once one time is better than 2 incremental changes